lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -321,7 +323,7 @@
                             headerActionsView: null, // TODO: add back SearchFieldView when search is enabled
                             title: options.title,
                             description: options.description,
-                            breadcrumbs: options.breadcrumbs
+                            breadcrumbs: this.createBreadcrumbs(topic)
```

Comment:
Nice catch! I've removed it and squashed, and I'll merge if I get a clean build.